strings.txt

```diff
@@ -540,6 +540,20 @@ PLUGIN_QOBUZ_SHOW_DISCS_DESC
 	NL	Heeft geen effect wanneer klassieke verbeteringen (zie hieronder) in gebruik zijn en er werken zijn geïdentificeerd
 	ES	No tiene efecto cuando se utilizan mejoras clásicas (ver más abajo) y se han identificado obras clásicas
 
+PLUGIN_QOBUZ_GROUP_RELEASES
```

Comment:
SETUP_GROUP_BY_RELEASE_TYPES_1

Reply:
What am I missing here? Are you suggesting changing the name of the string from "PLUGIN_QOBUZ_GROUP_RELEASES" to "SETUP_GROUP_BY_RELEASE_TYPES_1"? If so, why? And shouldn't it start with "PLUGIN_QOBUZ_" for "consistency"?   ;-)
Maybe it's the meds but I'm confused...

Reply:
What I wanted to suggest is to use that string as a base, to keep the wording consistent (see eg. the German version, which differs considerably).